queue/driver.go

```diff
@@ -48,6 +51,17 @@ type MongoDBOptions struct {
 	// field. If set to zero, the TTL index will not be created and
 	// and documents may live forever in the database.
 	TTL time.Duration
+	// LockTimeout overrides the default job lock timeout if set.
+	LockTimeout time.Duration
+}
+
+// GetLockTimeout returns the lock timeout specified by the options, or
+// amboy.LockTimeout as the default.
+func (opts *MongoDBOptions) GetLockTimeout() time.Duration {
```

Comment:
I might omit this method and just put this value in the DefaultMongoDBOptions constructor, and potentially have a validation method on MongoDBOptions, but not actually force it to be correct here.

Reply:
Done.